integration-tests/liquibase/src/test/java/io/quarkus/it/liquibase/LiquibaseFunctionalityTest.java

```diff
@@ -18,6 +18,20 @@ public void testLiquibaseQuarkusFunctionality() {
         doTestLiquibaseQuarkusFunctionality(isIncludeAllExpectedToWork());
     }
 
+    @Test
+    @DisplayName("Migrates a schema correctly using dedicated username and password from config properties")
+    public void testLiquibaseUsingDedicatedUsernameAndPassword() {
+        when().get("/liquibase/updateWithDedicatedUser").then().body(is(
+                "create-quarkus-table,insert-into-quarkus-table"));
+    }
+
+    @Test
+    @DisplayName("Checks the user")
```

Comment:
@gastaldi question. This test will only succeed if the `testLiquibaseUsingDedicatedUsernameAndPassword` is succeeded. So there is a dependency. Is that allowed?

Reply:
How about merging the checks with the test above?